main.py

```diff
@@ -111,13 +111,15 @@ def run_development_server(self) -> None:
     def _make_matrix_layout(self, path: str) -> html.Div:
         metric = routing.decode_matrix_path(path)
         results = self._cached_repo.get_mesh_results_all_connections()
+        config = self._cached_repo.get_mesh_config()
```

Comment:
_Non-critical:_ The variable should be called `mesh_config` in order to avoid confusion with app config (`self._config`). We should give `MeshResult` reference to `MeshConfig` if most consumers actually need to have access to it (as opposed to passing it separately).